### ISSUE TITLE
[FEAT]: Accessibility TTS on messages

### ISSUE THREAD

**Issue by timothycarambat** [Mintplex-Labs/anything-llm#1353]
### What would you like to see?

Enable the ability for browser-based TTS accessibility on messages. This should be a preference the user can enable that we can then improve the experience for as time goes on.

For now, if enabled, we should show a speaker icon that can be aria-attribute clicked that will read the ai message response using the default browser/machine TTS

**Comment by timothycarambat** [comment]
https://github.com/Mintplex-Labs/anything-llm/pull/909#issuecomment-2105087420

**Comment by crazyrobinhood** [comment]
That would be a great feature. Considering addind the coqui-tts env 
https://github.com/coqui-ai/TTS

**Comment by timothycarambat** [comment]
This feature would only be purely for accessibility and will not involve any external 3rd party providers right now as it is additional setup that will very likely go unused and the primary objective here is just to enhance accessibility of messages for those who are visually impaired or otherwise unable to read the message via screenreader
